pkg/geodata/rowquery_test.go

```diff
@@ -0,0 +1,80 @@
+// +build comptest
+
+package geodata
+
+import (
+	"context"
+	"flag"
+	"log"
+	"testing"
+
+	"github.com/ONSdigital/dp-find-insights-poc-api/comptests"
+	"github.com/ONSdigital/dp-find-insights-poc-api/model"
+	"github.com/ONSdigital/dp-find-insights-poc-api/pkg/database"
+)
+
+const dsn = "postgres://insights:insights@localhost:54323/censustest"
+
+// passing -args -kill=false to the test will kill docker postgres
```

Comment:
should this say `-kill=true` will kill docker postgres?

Reply:
The syntax is a bit odd. I'll add a makefile target which kills docker after a run.

Reply:
test-comptest-kill target